src/mon/ConfigKeyService.h

```diff
@@ -79,7 +79,7 @@ class ConfigKeyService : public QuorumService
   string get_name() const override {
     return "config_key";
   }
-  virtual void get_store_prefixes(set<string>& s);
+  void get_store_prefixes(set<string>& s) const;
```

Comment:
removing `virtual`, because there is no derived class of `ConfigKeyService` yet. i am even tempted to add `final`...